app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -1199,21 +1214,54 @@ class SamplesHeatmapView extends React.Component {
     );
   }
 
+  renderFilteredMultiplePipelineVersionsWarning(onClose, versions) {
+    return (
+      <Notification type={"warn"} displayStyle={"elevated"} onClose={onClose}>
+        <div>
+          <span className={cs.highlight}>
+            The selected samples come from multiple major pipeline versions:{" "}
+            {versions.join(", ")}.
```

Comment:
nit: I think there's a space missing after this sentence

Reply:
I would add `Contact us for help` with a link at the end of this message, because I don't believe users can currently re-run samples on their own.